pdns/dnsdist-lua-actions.cc

```diff
@@ -242,37 +243,48 @@ std::map<std::string,double> TeeAction::getStats() const
 void TeeAction::worker()
 {
   setThreadName("dnsdist/TeeWork");
-  char packet[1500];
-  int res=0;
-  struct dnsheader* dh=(struct dnsheader*)packet;
-  for(;;) {
-    res=waitForData(d_fd, 0, 250000);
-    if(d_pleaseQuit)
+  std::array<char, 1500> packet;
```

Comment:
I know this was already there, but I wonder if 1500 should be a constant somewhere

(I see how this code is safe, to be clear, with `sizeof()` before and `.size()` now)

Reply:
Good one, will do!